sqlglot/parser.py

```diff
@@ -1449,6 +1449,9 @@ def extend_props(temp_props: t.Optional[exp.Properties]) -> None:
                     exp.Clone, this=self._parse_table(schema=True), shallow=shallow, copy=copy
                 )
 
+        if self._curr:
```

Comment:
test case?

Reply:
sure, I think there are a couple more props missing based on https://github.com/tobymao/sqlglot/issues/2872, let's see if this picks them up:

> Unrelated, but STRICT/RETURNS NULL ON NULL INPUT also fails, thus I omitted it from the example above.

Reply:
https://github.com/tobymao/sqlglot/pull/2875/commits/4b441d2348014c41b575175350a6c061712fbc21